x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -97,6 +112,47 @@ export const createSignalsFieldAliases = () => {
   return fieldAliases;
 };
 
+export const backwardsCompatibilityMappings = [
+  {
+    minVersion: 0,
+    // Version 45 shipped with 7.14
+    maxVersion: 45,
+    mapping: {
+      runtime: {
+        'host.os.name.caseless': {
```

Comment:
Do we wanna move this out into its own json file or const just so we can add future fields without mixing all the data and logic together?

Reply:
We shouldn't be adding any future fields within this object, any future migrations would be separate objects at the top level of this array. And right now the field aliases are built dynamically from another JSON file, so we'd have to rework that logic to create the aliases at build time. It could be nice, but right now it's covered with a unit test against a snapshot.